script/setup.js

```diff
@@ -37,37 +36,48 @@ try {
 		strict: false,
 	});
 
-	async function getPrefillOrPromptedValue(key, message) {
+	async function getPrefillOrPromptedValue(key, message, placeholder) {
 		const { [key]: value } = values[key]
```

Comment:
So I tried this change out locally and getting an input doesn't seem to work?

```plaintext
Welcome to the template-typescript-node-package package setup!
This setup script will hydrate your repository based on your provided settings.

│
■  What will the kebab-case name of the repository be?
│
■  What will the Title Case title of the repository be?
│
■  What owner or user will the repository be under?
│
■  How would you describe the new package?
│
■  Whether to skip calling the GitHub API (effectively making this a local-only change).
│
```

I tried adding a `console.log({ value });` just before this `if` and it printed:

```plaintext
Welcome to the template-typescript-node-package package setup!
This setup script will hydrate your repository based on your provided settings.

│
■  What will the kebab-case name of the repository be?
│
{ value: undefined }
│
■  What will the Title Case title of the repository be?
│
{ value: undefined }
│
■  What owner or user will the repository be under?
│
{ value: undefined }
│
■  How would you describe the new package?
│
{ value: 'clack:cancel' }
│
■  Whether to skip calling the GitHub API (effectively making this a local-only change).
│
{ value: undefined }
```

I'm pretty sure the return from `await text` is the value itself, not an object with `[key]: value`. 

```suggestion
		const value = values[key]
```

Does that change work on your end @emday4prez?

Btw, the way I'm running this script is with `pnpm run setup --skip-api`.

Reply:
Yes that change works for me, I am still receiving the all-contributors error 
```
Clearing CHANGELOG.md...
✔️ Done.

Generating all-contributors table in README.md...
Error: 
    at file:///Volumes/SSD/dev%20projects/open-source/forked/template-typescript-node-package/script/setup.js:180:9
    exit code: 1
    at ChildProcess.<anonymous> (file:///Users/Emerson/.npm/_npx/721a1717bb84573e/node_modules/zx/build/core.js:146:26)
    at ChildProcess.emit (node:events:513:28)
    at ChildProcess.emit (node:domain:489:12)
    at maybeClose (node:internal/child_process:1091:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:513:28)
    at Socket.emit (node:domain:489:12)
    at Pipe.<anonymous> (node:net:322:12)
    at Pipe.callbackTrampoline (node:internal/async_hooks:130:17)

Removing devDependency packages only used for setup...
✔️ Done.

Looks like there was a problem. Correct it and try again? 😕
```